spec/integrations/rebound_bracket_spec.rb

```diff
@@ -0,0 +1,14 @@
+describe 'rebinding [' do
+  context 'initialized before sourcing the plugin' do
+    before do
+      session.send_string("function [ { $commands[\\[] \"$@\" }")
+      session.send_keys("'enter'")
```

Comment:
You should be able to use `session.run_command` here instead of both `send_string` and `send_keys`.

https://github.com/zsh-users/zsh-autosuggestions/blob/master/spec/terminal_session.rb#L25-L30